### ISSUE TITLE
Android 11: "Sorry, no external application has been found to complete this action" when trying to open a PDF

### ISSUE THREAD

**Issue by ThiefMaster** [vector-im/element-android#3067]
#### Describe the bug
When tapping a PDF file someone sent me (or I sent), it doesn't open, but `error_no_external_application_found` ("Sorry, no external application has been found to complete this action.") shows up instead. The only workaround right now is to long-press the file, save locally, and then open it from the file download notification or a file manager.

#### To Reproduce
1. Receive a PDF (maybe sending one yourself works as well, I tested it with one I sent via the desktop app)
2. Click on the file
3. See error

#### Expected behavior
The PDF file opening in any PDF viewer on the phone

#### Smartphone (please complete the following information):
 - Device: Pixel 5 (happens on Pixel 3a XL as well)
 - OS: Android 11

#### Additional context
 - App version and store: 1.1.3 via Play store
 - Homeserver: private (ran by me, so if it might be homeserver-related (which I doubt) let me know)

**Comment by warwicklol14** [comment]
I had the same bug too on my Samsung M31 device running Android 11. It was not just for pdf but also for pptx, docx files. 
I tried to find what was causing the bug by debugging it. 

It turns out that `intent.resolveActivity(requireActivity().packageManager)` in `openFileIntent()` of `RoomDetailFragment.kt` always returns `null` regardless of the file in Android 11.

I searched some more and found that package visibility changed in Android 11 https://medium.com/androiddevelopers/package-visibility-in-android-11-cc857f221cd9
It says here in https://developer.android.com/training/package-visibility/use-cases#kotlin that it's better to handle the ActivityNotFoundException.

So I tested it and it works fine after this change https://github.com/warwicklol14/element-android/commit/a6e2947115c698e56b796be04e7f8631c3f49de0.

Tested on:
    Android 11 - Android Emulator (Pixel 3a) , Real device (Samsung M31)
    Android 10 - Android Emulator (Pixel 2)
    Android API 21 - Android Emulator (Pixel 2) (As requested in CONTRIBUTING.md)

All lints and quality checks pass.

I would like to make a pull request if the code is okay with the element android team.
